contrib/ansible/roles/skydive_agent/tasks/config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Clean-up neutron probe section
+  yedit:
+    src: "{{ skydive_config_file }}"
+    key: "agent.topology.neutron"
+    state: absent
+
+- name: Setup neutron probe section
+  yedit:
+    src: "{{ skydive_config_file }}"
+    edits:
+        - key: "agent.topology.neutron.auth_url"
```

Comment:
openstack.auth_url
:)

Reply:
not yet as the stable version is not using the new params, need to wait a bit before  changing here as well

Reply:
I mean packages, etc...